src/components/slider/slider.scss

```
@@ -7,11 +7,18 @@
 @use '@material/slider/styles';
 @use '@material/floating-label/mdc-floating-label';
 
+:host([disabled]) {
```

With the above, there's a difference if the component is `readonly` _and_ `disabled`, or _just_ `readonly`:

https://user-images.githubusercontent.com/154725/151988254-56f37344-e5ae-4813-bb3d-5079bc0d11e1.mov

It's preferable if the readonly state looks the same whether or not the component also has the `disabled` attribute set:

```suggestion
:host([disabled]:not([readonly])) {
```